server.js

```diff
@@ -27,6 +17,123 @@ app.get('/', (req, res) => {
   res.send('Hello world')
 })
 
+
+//GETS THE WHOLE DATA-SET UNFILTERED
+app.get('/collection', (req, res) => {
+  res.json(netflixData)
+})
+
+
+//GETS ALL MOVIES
+app.get('/movies', (req, res) => {
+  const onlyMovies = netflixData.filter((item) => item.type === "Movie")
+
+  res.json(onlyMovies)
+})
+
+
+//LOOKS FOR ALL MOVIES FROM A SPECIFIC YEAR AND ADDITIONAL QUERY FOR COUNTRY
+app.get('/movies/:year', (req, res) => {
+  const year = req.params.year
+  const onlyMovies = netflixData.filter((item) => item.type === "Movie")
+  let moviesFromYear = onlyMovies.filter((item) => item.release_year === +year)
+  const country = req.query.country
+
+  if (country) {
+    console.log(country)
+    moviesFromYear = moviesFromYear.filter((item) => item.country.toLowerCase() === country.toLowerCase())
+  }
+
+  res.json(moviesFromYear)
+})
```

Comment:
Path params are mostly used to identify a specific resource, so I think it would be better to have this as a query param, since you're filtering here. Like this: `/movies?year=2019`